include/industry_standard/spdm.h

```diff
@@ -403,13 +416,6 @@ typedef struct {
 	uint8 nonce[32];
 } spdm_challenge_request_t;
 
-///

```

Comment:
This change was just to move these defines after the response struct so I could wrap both the request/response with a single #if/#endif.